docs/index.html

```diff
@@ -476,7 +476,7 @@ <h2 id="html-support">HTML support</h2>
 <p>It is generally recommended to enter the notes as Markdown as it makes the notes easier to edit. However for cases where certain features aren&#39;t supported (such as strikethrough or to highlight text), you can also use HTML code directly. For example this would be a valid note:</p>
 <pre><code>This is &lt;s&gt;strikethrough text&lt;/s&gt; mixed with regular **Markdown**.
 </code></pre><h2 id="custom-css">Custom CSS</h2>
-<p>Rendered markdown can be customized by placing a userstyle file in the profile directory <code>~/.config/joplin-desktop/userstyle.css</code> (This path might be different on your device - check at the top of the Config screen for the exact path). This file supports standard CSS syntax.</p>
+<p>Rendered markdown can be customized by placing a userstyle file in the profile directory <code>~/.config/joplin-desktop/userstyle.css</code> (This path might be different on your device - check at the top of the Config screen for the exact path). This file supports standard CSS syntax. To handle exporting of PDF fils with hilighting, add <code>@media print { ... }</code> and put the rules you need</p>
```

Comment:
I suggest a few changes to the sentence (spelling and a minor rewording):

To handle the export of PDF files with highlighting, add <code>@media print { ... }</code> and add the rules you need.

Reply:
Hi @foxmask .

I think you missed my other changes:

it's files, not fils
it's highlighting, not hilighting

The start of the sentence should also be rewritten. It should either read: 
- `To handle the export to PDF files...`
- `For exporting PDF files...`.

Reply:
Oh, are you sure that the index.html file is not generated by another file? Just asking, because in some project the documentation is generated and if it's not changed in the source, the updates would be overwritten the next time it's run.

Reply:
I don't know. 
All the build tools I know are for generating the application itself.  when I look at https://github.com/laurent22/joplin/commit/f781cb3922e00b0a271427c5c1e404e728e78fef I see what changed but I don't see if a tool get the version from somewhere and spread the version on every files it needs

Reply:
@foxmask I'm pretty sure the index.html is just built from the README (in build-website.js)

https://github.com/laurent22/joplin/blob/1d4234caeab1f7736381d785bbb4ea9bef4acbf6/Tools/build-website.js#L378

Reply:
Thanks.
I will redo the pr from scratch. And drop this one.

Reply:
You can always squash your commits, but I think that the README also has to be changed.

Other than that the sentence is ok now. The only thing missing is the period at the end of the sentence.